include/ktx.h

```diff
@@ -1291,7 +1291,9 @@ typedef struct ktxBasisParams {
              selector RDO, no endpoint RDO). Only valid for linear textures.
          */
     ktx_bool_t separateRGToRGB_A;
-        /*!< Separates the input R and G channels to RGB and A (for tangent
+        /*!< @deprecated. Use inputSwizzle "rrrg" instead. If texture has 2 components
+             this swizzle will be automatically applied.
+             Separates the input R and G channels to RGB and A (for tangent
```

Comment:
Delete this line and the rest of the documentation for separateRGToRGB_A.

Reply:
https://github.com/KhronosGroup/KTX-Software/pull/493/commits/4c52f30f072c1b351ed04d6ae7a57c558074acd2